src/components/select/select-list/select-list.component.js

```diff
@@ -334,9 +339,38 @@ const SelectList = React.forwardRef(
       );
     }
 
+    const loader = () => (
+      <StyledSelectLoaderContainer key="loader" as={multiColumn ? "div" : "li"}>
+        <Loader />
+      </StyledSelectLoaderContainer>
+    );
+
+    let selectListContent;
```

Comment:
Might as well just 
``` 
let selectListContent = (
   <>
      {childrenWithListProps}
      {isLoading && loader()}
   </>
);
``` 
Then do `if (multiColumn) ...` and then we don't need the `else`